common/common.scss

```diff
@@ -212,3 +212,37 @@ input {
 .alert-emails-disabled {
     display: none;
 }
+
+.badge-card {
+  background-color: #FFFFFF;
+  border: 1px solid #E9EAEE;
+  
+  .medium:hover {
+    box-shadow: 0px 6px 12px rgba(0, 0, 0, 0.083);
+  }
+
+  .grant-count {
+    right: unset;
+    left: 10%;
+    top: 16px;
+    color: #E9EAEE;
+    font-family: 'Helvetica Neue', Arial, sans-serif;
+    font-weight: normal;
+    font-size: 4.5em;
+  }
+
+  .badge-contents {
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 24px 10%;
+    .badge-icon {
+      width: 100%;
+      justify-content: flex-end;
+      margin-bottom: -1rem;
```

Comment:
It seems to be unusual that using a negative margin. If we'd like to remove gaps between `.badge-icon` and `.badge-info`, I will suggest removing the padding value in line 244.